src/hooks/useNodeJsContributorsApi.tsx

```diff
@@ -54,19 +55,34 @@ async function getMaxContributors(): Promise<[number, number]> {
 
 /**
  * Retrieves a contributor's object by it's index in API
+ * It stores contributors in localStorage in order to do less consequent requests
  * @param randomPage
  */
 async function getContributor(randomPage: number): Promise<Contributor> {
-  const response = await fetch(`${CONTRIBUTORS_API_URI}&page=${randomPage}`);
-  const contributor = (await response.json())[0] as ContributorApiResponse;
-
-  return {
-    avatarUri: contributor.avatar_url,
-    commitsListUri: `https://github.com/nodejs/node/commits?author=${contributor.login}`,
-    contributionsCount: contributor.contributions,
-    login: contributor.login,
-    profileUri: contributor.html_url,
-  };
+  const response = (await (
```

Comment:
I feel instead of using `await` here, we could use Promise chaining. It fits better in this case, as this whole function returns a Promise.